axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConfiguration.java

```diff
@@ -151,6 +151,12 @@ public class AxonServerConfiguration {
      */
     private FlowControlConfiguration commandFlowControl;
 
+    /**
+     * A toggle dictating whether to invoke query handlers directly if they are registered in the local environment.
+     * Defaults to {@code false}.
+     */
+    private boolean localSegmentShortCut = false;
```

Comment:
Given the rename on the getter, I would rename the field in a similar fashion.
So, to `shortcutQueriesToLocalHandlers`:
```suggestion
    private boolean shortcutQueriesToLocalHandlers = false;
```